docs/readme_sections/mapping_api.rst

```diff
@@ -422,6 +422,72 @@ subject type ``line`` with different edge types. The cell values
 values ``sensitivity`` and ``productivity`` would be mapped to the node
 type ``noun`` via the edge type ``line_is_noun``.
 
+Type branching based on value from another column
```

Comment:
We should not have duplicated documentation. If it's in the doc, you can safely discard what's in the README above.